tf2_geometry_msgs/include/tf2_geometry_msgs/tf2_geometry_msgs.hpp

```diff
@@ -347,6 +349,104 @@ template <>
 inline
   std::array<std::array<double, 6>, 6> getCovarianceMatrix(const geometry_msgs::msg::PoseWithCovarianceStamped& t)  {return covarianceRowMajorToNested(t.pose.covariance);}
 
+// Forward declaration
+void fromMsg(const geometry_msgs::msg::Transform& in, tf2::Transform& out);
+
+/** \brief Transform the covariance matrix of a PoseWithCovariance message to a new frame.
+ * \param cov_in The covariance matrix to transform.
+ * \param transform The transform to apply, as a Transform message.
+ * \return The transformed covariance matrix.
+ */
+inline
+geometry_msgs::msg::PoseWithCovariance::_covariance_type transformCovariance(const geometry_msgs::msg::PoseWithCovariance::_covariance_type& cov_in, const geometry_msgs::msg::Transform& transform)
```

Comment:
It looks like this file isn't being linted, but I'd still recommend [limiting the line length to 100 characters](https://docs.ros.org/en/rolling/Contributing/Code-Style-Language-Versions.html#line-length) for readability.

Reply:
This is what `ament_uncrustify` recommends as well, so I will do this.

```suggestion
geometry_msgs::msg::PoseWithCovariance::_covariance_type transformCovariance(
  const geometry_msgs::msg::PoseWithCovariance::_covariance_type & cov_in,
  const geometry_msgs::msg::Transform & transform)
```

Reply:
Addressed in 8545842360dad058